db/schema.rb

```diff
@@ -28,11 +26,11 @@
 
   create_table "application_letters", force: :cascade do |t|
     t.string   "motivation"
-    t.integer  "user_id",                      null: false
-    t.integer  "event_id",                     null: false
-    t.datetime "created_at",                   null: false
-    t.datetime "updated_at",                   null: false
-    t.integer  "status",           default: 2, null: false
+    t.integer  "user_id",                               null: false
+    t.integer  "event_id",                              null: false
+    t.datetime "created_at",                            null: false
+    t.datetime "updated_at",                            null: false
+    t.integer  "status",                    default: 2, null: false
```

Comment:
I get changes like this almost all the time, I guess it's related to how different versions of ruby or rails decide how many tabs are needed.